src/common/api/types.ts

```diff
@@ -1253,10 +1253,8 @@ export interface HostUpdateParams {
    * A string which will be used as Authorization Bearer token to fetch the ignition from ignitionEndpointUrl.
    */
   ignitionEndpointToken?: string;
-  /**
-   * Labels to be added to the corresponding node.
-   */
-  nodeLabels?: NodeLabelParams[];
+  validationsInfo?: string;
+  nodeLabels?: string;
```

Comment:
```suggestion
  validationsInfo?: string;
  nodeLabels?: string;
```

Where are these fields coming from? Are they present in swagger.yaml as well? I don't see the swagger.yaml modified here...

Reply:
You are right, I don't think it should be included for now.
I reverted it and we can add it with all its required changes it when Node labelling is implemented.